src/main/java/org/jenkinsci/plugins/proccleaner/ProcCleaner.java

```diff
@@ -60,6 +60,19 @@ public static int getpid(){
     }
 
     /**
+     *
+     * @return OS name with lowered cases
+     */
+    public static String getOsName() {
+        String os = System.getProperty("os.name", "");
+
+        if (os.length() > 0)
```

Comment:
Oliver, your are right, the condition isn't necessary in current impl., fixing...

Pavel